src/realtime/ash/ash_hazard.py

```diff
@@ -130,82 +159,49 @@ def copy_style(self):
-                            u'very low': [0.01, 0.1],
-                            u'medium': [2.0, 5.0],
-                            u'low': [0.1, 2.0]
-                        }
-                    }
-                }, u'population': {
-                    u'ash_hazard_classes': {
-                        u'active': True, u'classes': {
-                            u'high': [5.0, 10.0],
-                            u'very high': [10.0, 9999.0],
-                            u'very low': [0.01, 0.1],
-                            u'medium': [2.0, 5.0],
-                            u'low': [0.1, 2.0]
-                        }
-                    }
-                }
+                exposure_structure['key']: hazard_classes,
+                exposure_place['key']: hazard_classes,
+                exposure_land_cover['key']: hazard_classes,
+                exposure_road['key']: hazard_classes,
+                exposure_population['key']: hazard_classes
             },
-            'keyword_version': u'4.3',
-            'hazard': u'volcanic_ash',
-            'continuous_hazard_unit': u'centimetres',
-            'layer_geometry': u'raster',
-            'layer_purpose': u'hazard',
-            'layer_mode': u'continuous',
+            'keyword_version': inasafe_keyword_version,
+            'hazard': hazard_volcanic_ash['key'],
+            'continuous_hazard_unit': unit_centimetres['key'],
+            'layer_geometry': layer_geometry_raster['key'],
+            'layer_purpose': layer_purpose_hazard['key'],
+            'layer_mode': layer_mode_continuous['key'],
             'extra_keywords': {
-                u'volcano_name': self.volcano_name,
-                u'region': self.region,
-                u'volcano_alert_level': self.alert_level,
-                u'time_zone': str(self.time_zone),
-                u'volcano_latitude': str(self.latitude),
-                u'volcano_longitude': str(self.longitude),
-                u'volcano_event_id': self.ash_id,
-                u'volcano_eruption_event_time': str(self.event_time),
-                u'volcano_eruption_height': str(self.eruption_height),
-                u'volcano_height': str(self.vent_height),
+                extra_keyword_volcano_name['key']: self.volcano_name,
+                extra_keyword_region['key']: self.region,
+                extra_keyword_volcano_alert_level['key']: self.alert_level,
+                extra_keyword_time_zone['key']: str(self.time_zone),
+                extra_keyword_volcano_latitude['key']: str(self.latitude),
+                extra_keyword_volcano_longitude['key']: str(self.longitude),
+                extra_keyword_volcano_event_id['key']: self.ash_id,
+                extra_keyword_volcano_eruption_event_time['key']: str(
+                    self.event_time),
+                extra_keyword_eruption_height['key']: str(self.eruption_height),
+                extra_keyword_volcano_height: str(self.vent_height),
```

Comment:
maybe typo here, you forgot the key

Reply:
Ah yes, thanks.... Too many copy paste :D